conf/app.routes

```diff
@@ -2,6 +2,6 @@
 
 GET        /show-results/vrn/:vrn        controllers.Controller.showResults(vrn : Vrn)
 GET        /view-repayment-account/:accountHolderName/:bankAccountNumber/:sortCode      controllers.Controller.viewRepaymentAccount(accountHolderName: AccountHolderName, bankAccountNumber: BankAccountNumber, sortCode : SortCode)
-
+GET        /manage-or-track/vrn/:vrn     controllers.Controller.manageOrTrack(vrn : Vrn)
```

Comment:
At some point we should refactor it and remove `:vrn` from url. It's incentive for users to force browse trying someones else vrn number.